package.json

```diff
@@ -1,23 +1,75 @@
 {
   "name": "@transcend-io/conflux",
-  "version": "5.0.0",
+  "version": "6.0.0",
+  "type": "module",
   "description": "Build zip files out of readable streams in the browser",
-  "main": "dist/conflux.umd.min.js",
-  "jsdelivr": "dist/conflux.umd.min.js",
-  "unpkg": "dist/conflux.umd.min.js",
-  "browser": "dist/conflux.esm.js",
-  "module": "dist/conflux.esm.js",
+  "main": "./dist/commonjs/index.js",
+  "module": "./dist/esm/index.js",
```

Comment:
I _think_ if you set `exports` you don't need to set these two fields, but I could be wrong?

Reply:
yeah this doesn't like not having `main`, but i'm pretty confident the other two ("types" and "module") are targeting things that will always know to read "exports" so i'll remove those

<img width="625" height="175" alt="Screenshot 2025-08-12 at 7 24 18 PM" src="https://github.com/user-attachments/assets/a8083aa9-169f-4ffb-887a-eb16cf8ec698" />